src/content/features/helxFunctions.mdx

```diff
@@ -0,0 +1,28 @@
+import { FunctionsGridItem } from "../../components/sections/functions.js";
+import { GridContainer } from "../../components/grid.js";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+
+<MaxWidthWrapper gutterBottom>
+
+## What can HeLx do for you?
+
+<GridContainer padding="1rem 0 0 0">
```

Comment:
i dont think you need the `padding` prop here. i think it looks alright without the top padding